---
#This file is autogenerated and therefore does not contain documentation of the parameters
#See the README in the config folder for descriptions on each parameter
robot name: 2segment
num segments: 2
sections per segment: 1
masses: [0.12, 0.02, 0.042000000000000003, 0.023]
lengths: [0.113, 0.029999999999999999, 0.125, 0.0050000000000000001]
diameters: [0.035000000000000003, 0.028000000000000001, 0.019800000000000002]
armAngle: 180
shear modulus: [78881.582904262934, 81977.953812553489]
drag coef: [28000, 8000]
valvemap: [1, 6, 4, 2, 3, 0, 5]
sensor refresh rate: 100
bendlabs address: /dev/ttyACM0
model update rate: 100
chamberConfigs: [-1, 0.5, 0.5, 0, 0.866, -0.866, -1, 0.5, 0.5, 0, 0.866, -0.866]
p_max: 650
prismatic: false
horizontal: true
sensors: [qualisys]
model type: augmented
